### ISSUE TITLE
[zgui] missing DrawList.addCallback

### ISSUE THREAD

**Issue by musi-musi** [michal-z/zig-gamedev#323]
Currently no binding present for ImDrawList::AddCallback()

**Comment by michal-z** [comment]
Thanks for the report, would like to make an PR?

**Comment by musi-musi** [comment]
Absolutely; I've just been busy with other parts of my project but ill have one in later
